src/main/java/picard/util/LiftoverUtils.java

```diff
@@ -89,7 +84,16 @@ public static VariantContext liftVariant(final VariantContext source, final Inte
 
         builder.filters(source.getFilters());
         builder.log10PError(source.getLog10PError());
-        builder.attributes(source.getAttributes());
```

Comment:
why did this get deleted?

Reply:
because it already happens earlier in the code. specifically in lines picard/util/LiftoverUtils.java:154 or picard/util/LiftoverUtils.java:140 depending on the path taken.

Reply:
(the constructor that takes a vc copies the attributes)

Reply:
thanks!